### PR TITLE
Modify PR template to encourage contribution of automated tests

### PR DESCRIPTION
**Pull request by dlenski**
Many recent PRs offer up shell scripts or example queries in the testing section, when these could be easily and usefully be converted to automated tests based on MTR.  (https://github.com/MariaDB/server/pull/1888 and https://github.com/MariaDB/server/pull/1940 are good examples of this.)

This modifies the PR template to more strongly encourage the contribution of
automated tests.

# Copyright
All new code of the whole pull request, including one or several files that are either new files or modified ones, are contributed under the BSD-new license. I am contributing on behalf of my employer Amazon Web Services, Inc.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,8 +26,15 @@ An amazing description should answer some questions like:
 TODO: fill description here
 
 ## How can this PR be tested?
-TODO: fill steps to reproduce here, if applicable,
-      or remove the section
+
+TODO: modify the automated test suite to verify that the PR causes MariaDB to
+behave as intended, using [mysql-test](https://mariadb.com/kb/en/mysql-test-overview).
```
Comment (robertbindar) [inline .github/pull_request_template.md:31]: Please replace link with "https://mariadb.org/get-involved/getting-started-for-developers/writing-good-test-cases-mariadb-server/". This small tutorial is less fluffy than mtr overview, just a better tool for the job.

**Review by robertbindar — APPROVED** [review]
Thanks for the PR @dlenski! I love you suggested a template adjustment based on behavior observed in other PRs using it, that's the way this template should be improved for sure!
One tiny comment about the link of MTR info, other than that good to merge.

File: .github/pull_request_template.md (hunk 1 above)
Comment (dlenski) [reply .github/pull_request_template.md]: Thanks, done :heavy_check_mark:

**Comment by robertbindar** [comment]
Pushed to 10.8, many thanks again @dlenski!

**Pull request closed without merging**